pkgs/by-name/te/terraform-plugin-docs/package.nix

```diff
@@ -19,10 +21,14 @@ buildGoModule rec {
 
   vendorHash = "sha256-AjW6BokLVDkIWXToJ7wNq/g19xKTAfpQ/gVlKCV5qw0=";
 
+  nativeBuildInputs = [ makeWrapper ];
+
   subPackages = [
     "cmd/tfplugindocs"
   ];
 
+  allowGoReference = true;
```

Comment:
Since the `tfpluginsdocs` binary is wrapped to add go in the PATH, do we really need this?

Reply:
Thanks for the review.

Yes we need to enable allowGoReference here, otherwise the build will fail with:

```console
error: output '/nix/store/pw904v0071lqxjkm1p7z0bmvvb161ap2-terraform-plugin-docs-0.16.0' is not allowed to refer to the following paths:
         /nix/store/dwmb0qcai52d0zkgpm6f5ifx2a8yvsdg-go-1.21.3
```

because the `buildGoModule` won't allow to reference the go compiler in the package.

And we need it to be in the PATH for the terraform-plugin-docs package.

Reply:
Ok, thanks.